src/validations/validation.ts

```diff
@@ -0,0 +1,29 @@
+export default class Validation {
+  public id = '';
+  public github = '';
+  public version = '';
+
+  public author: string;
+  public space: string;
+  public network: string;
+  public snapshot: number | 'latest';
+  public params: any;
+
+  constructor(
+    author: string,
+    space: string,
+    network: string,
+    snapshot: number | 'latest',
+    params: any
+  ) {
+    this.author = author;
+    this.space = space;
+    this.network = network;
+    this.snapshot = snapshot;
+    this.params = params;
+  }
+
+  async validate(): Promise<boolean> {
+    return true;
+  }
```

Comment:
Why do we need this? 🤔 it always return true?

Reply:
Its just to set the class format